diff --git a/node_modules/react-native-safe-area-context/src/SafeAreaContext.tsx b/node_modules/react-native-safe-area-context/src/SafeAreaContext.tsx
index c06573a..22e4af7 100644
--- a/node_modules/react-native-safe-area-context/src/SafeAreaContext.tsx
+++ b/node_modules/react-native-safe-area-context/src/SafeAreaContext.tsx
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { Dimensions, StyleProp, StyleSheet, ViewStyle } from 'react-native';
+import { Dimensions, StyleProp, StyleSheet, ViewStyle, Platform, StatusBar } from 'react-native';
 import NativeSafeAreaProvider from './NativeSafeAreaProvider';
 import { EdgeInsets, InsetChangedEvent, Metrics, Rect } from './SafeArea.types';
 
@@ -70,6 +70,10 @@ export function SafeAreaProvider({
     [frame, insets],
   );
 
+  if (insets != null && Platform.OS === 'android' && StatusBar.currentHeight !== undefined) {
+    insets.top = StatusBar.currentHeight;
+  }
+
   return (
     <NativeSafeAreaProvider
       style={[styles.fill, style]}
